.github/workflows/build.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+name: Build wheels
+on: push
+jobs:
+  manylinux_wheels:
+    name: Python ${{ matrix.python_tag }} ${{ matrix.platform }} wheels
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform:
+          - manylinux1_x86_64
+          - manylinux2014_x86_64
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Docker
+        run: sudo apt-get install -y --no-install-recommends docker
+      - name: Pull Docker image
+        run: |
+          DOCKER_IMAGE="quay.io/pypa/${{ matrix.platform }}"
+          echo "::set-env name=DOCKER_IMAGE::$DOCKER_IMAGE"
+          docker image pull "$DOCKER_IMAGE"
+      - name: Build manylinux wheels
+        env:
+          PYTHON_TAGS: 'cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38'
+        run: |
+          docker container run \
+            --rm \
+            -e PLAT=${{ matrix.platform }} \
+            -e PYTHON_TAGS="$PYTHON_TAGS" \
+            -v "$(pwd)":/io \
+            --workdir '/io' \
+            "$DOCKER_IMAGE" \
+            /io/packaging/build/manylinux.sh
```

Comment:
Only owing to my inexperience with GHA and what is available there. To be clear, are you talking about something like `uses:` to enable Docker rather than installing it via package manager?

Reply:
No, I was talking about something a bit more native: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer.